crates/codegen/parser/runtime/src/templates/queries.rs.jinja2

```diff
@@ -0,0 +1,56 @@
+use std::sync::Once;
+use crate::generated::query::Query;
+
+mod __query_kind {
```

Comment:
IIUC, we wrap the enum in a `__query_kind` sub-module, then immediately expose it outside. Is this needed?

Reply:
Yes, this is a hack suggested by Igor to deal with a linting problem with `napi(string_enum, namespace = "queries")`

Reply:
done